src/app/views/dashboard/account/account.vue

```diff
@@ -85,6 +85,14 @@
 										<translate>Password</translate>
 									</router-link>
 								</li>
+								<li v-if="shouldShowVerify">
+									<router-link
+										:to="{ name: 'dash.account.verified-account' }"
+										active-class="active"
+									>
+										<translate>Verify Account</translate>
```

Comment:
Couldn't this be confused with that annoying step or verifying one's account through the email link? How else can we call it to keep it short and clear?

Reply:
Maybe "Verification Application"?